effects.cwt

```diff
@@ -2944,3 +2964,14 @@ alias[effect:set_looter_hostility_days] = {
     who = scope[character]
     days = int
 }
+
+## scope = character
+### True if the scoped character has a strong claim on the given title.
+alias[effect:has_strong_claim] = scope[title]
```

Comment:
Uh, shouldn't that be in triggers? And I think there's already a trigger for that.

Reply:
Yh. Done.